### PR TITLE
Update page confirmation popup

### PR DESCRIPTION
**Pull request by rodaserg0208**
## Describe Your Changes
Added the confirmation popup in update-save-button.component.ts. Made tests for it in the test file for that popup. Also made the test for the pop up in the save-button.component. 

## Type of Change
- [x] New feature (change that adds functionality)
- [x] Implemented tests

## Checklist (DOD)
- [x] The code is written and passes all Acceptance criteria's
- [ ] Any Zombie code is removed
- [ ] Code has proper comment documentation
- [x] The code is unit tested with happy case and passes
- [x] Code is refactored for optimization
- [ ] The code has been peer-reviewed by one or more team members who did not work on it.
- [x] The code has no merge conflicts

### If Applicable Checklist
- [ ] The code is integration tested and passes (if applicable)
- [ ] The code is regression tested and passes (if applicable)
- [ ] Code is reviewed and accepted by Product Owner (if applicable)

### CODE REVIEW

File: front-end/src/custom-theme.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (sophialhales) [inline front-end/src/custom-theme.scss:1]: Delete whitespace

File: front-end/src/custom-theme.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+// (imported above). For each palette, you can optionally specify a default, lighter, and darker
+// hue. Available color palettes: https://material.io/design/color/
+$front-end-primary: mat.define-palette(mat.$indigo-palette);
+$front-end-accent: mat.define-palette(mat.$pink-palette, A200, A100, A400);
+
+// The warn palette is optional (defaults to red).
+$front-end-warn: mat.define-palette(mat.$red-palette);
+
+// Create the theme object. A theme consists of configurations for individual
+// theming systems such as "color" or "typography".
```
Comment (sophialhales) [inline front-end/src/custom-theme.scss:22]: Would rewrite comment to be more relevant to our project, or delete it

File: front-end/src/custom-theme.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+// Plus imports for other components in your app.
+
+// Include the common styles for Angular Material. We include this here so that you only
+// have to load a single css file for Angular Material in your app.
+// Be sure that you only ever include this mixin once!
+@include mat.core();
+
+// Define the palettes for your theme using the Material Design palettes available in palette.scss
+// (imported above). For each palette, you can optionally specify a default, lighter, and darker
+// hue. Available color palettes: https://material.io/design/color/
```
Comment (sophialhales) [inline front-end/src/custom-theme.scss:14]: Would rewrite comment to be more relevant to our project, or delete it

File: front-end/src/custom-theme.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+
+// Custom Theming for Angular Material
+// For more information: https://material.angular.io/guide/theming
+@use '@angular/material' as mat;
+// Plus imports for other components in your app.
+
+// Include the common styles for Angular Material. We include this here so that you only
+// have to load a single css file for Angular Material in your app.
+// Be sure that you only ever include this mixin once!
```
Comment (sophialhales) [inline front-end/src/custom-theme.scss:9]: Would rewrite comment to be more relevant to our project, or delete it

File: front-end/src/custom-theme.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  color: (
+    primary: $front-end-primary,
+    accent: $front-end-accent,
+    warn: $front-end-warn,
+  )
+));
+
+// Include theme styles for core and each component used in your app.
+// Alternatively, you can import and @include the theme mixins for each component
+// that you are using.
```
Comment (sophialhales) [inline front-end/src/custom-theme.scss:33]: Would rewrite comment to be more relevant to our project, or delete it

File: front-end/src/custom-theme.scss [hunk 6]
```diff
@@ -0,0 +1,35 @@
+
+// Custom Theming for Angular Material
+// For more information: https://material.angular.io/guide/theming
```
Comment (sophialhales) [inline front-end/src/custom-theme.scss:3]: can be condensed down to single line

**Review by sophialhales — CHANGES_REQUESTED** [review]
Save changes popup actually pops up in Karma, need to mock it instead

**Comment by ColeNelson806** [comment]
> Save changes popup actually pops up in Karma, need to mock it instead

Should be fixed now

**Review by sophialhales — APPROVED** [review]
looks good to me!

**Pull request merged**